tests/ClientTestCase.php

```diff
@@ -6,11 +6,19 @@
 use Keboola\StorageApi\Client;
 use Keboola\Test\ClientProvider\TestSetupHelper;
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
 
 class ClientTestCase extends TestCase
 {
     use \PHPUnitRetry\RetryTrait;
 
+    public function getLogger(): ConsoleLogger
+    {
+        return new ConsoleLogger(new ConsoleOutput(OutputInterface::VERBOSITY_DEBUG));
```

Comment:
```suggestion
        return new ConsoleLogger(new ConsoleOutput(OutputInterface::VERBOSITY_VERY_VERBOSE));
```

Reply:
Protože to bleje hromadu spamu, který reálně nechceš a pak v tom nenajdeš ty warningy